plugins/woocommerce-admin/client/task-lists/fills/tax/index.tsx

```diff
@@ -162,6 +163,7 @@ const Tax: React.FC< TaxProps > = ( { onComplete, query, task } ) => {
 				component: WooCommerceTax,
 				isVisible:
 					! taxJarActivated && // WCS integration doesn't work with the official TaxJar plugin.
+					! woocommerceTaxActivated &&
```

Comment:
Makes the WC > Home > task list > Collect sales tax task display this:

<img width="742" alt="Screenshot 2024-06-06 at 14 00 11" src="https://github.com/woocommerce/woocommerce/assets/1759681/5ff7489e-843f-4bc9-a9d4-495ac809f41e">

(The "Configure" button leads to the WC > Settings > Tax > Standard rates page.)

Instead of this:

<img width="740" alt="Screenshot 2024-06-06 at 13 59 50" src="https://github.com/woocommerce/woocommerce/assets/1759681/0132a9e6-2ca5-4c72-b364-51c1909439d5">